### ISSUE TITLE
PHP 5.3 version?

### ISSUE THREAD

**Issue by dwhalenenr** [thephpleague/oauth2-client#661]
Hi.  I'm just wondering if this package is safe with PHP 5.3.  We're struggling to find a good OAuth2.0 library for LinkedIn and I hoped this might do the trick....

**Comment by peterdd** [comment]
0.12.1 seems to the last official version supporting < php5.5 : 

> https://github.com/thephpleague/oauth2-client/releases?after=1.0.0
> BREAK: Requires PHP 5.5.0 and greater.

**Comment by ramsey** [comment]
This package will not work with PHP 5.3. I'm sorry. It requires language features that were introduced in later versions of PHP.

**Comment by peterdd** [comment]
Ah, I remember 0.12.1 php5.4+. 0.3 for php5.3 with this typo filename fix:

```
 # fix for oauth2-client v0.3 ( the last version that works with php5.3.* )
 mv vendor/league/oauth2-client/src/Grant/AuthorizationCode.php vendor/league/oauth2-client/src/Grant/Authorizationcode.php
```
